.swiftformat

```diff
@@ -0,0 +1,24 @@
+# file options
```

Comment:
In all of our recent projects, we actually adopted `swift-format` since it has a more stable formatting between versions. Would recommend picking it here as well.

Reply:
Can we change this in a follow up?